admin/pre_reg/serializers.py

```diff
@@ -21,18 +21,16 @@ def serialize_user(user):
 
 # TODO: Write and use APIv2 serializer for this
 def serialize_draft_registration(draft, json_safe=True):
-    if draft.branched_from is not None:
-        url = furl(OSF_DOMAIN)
-        url.path.add(draft.branched_from.url)
-        node_url = url.url
-    else:
-        node_url = None
-    registration_choice = draft.approval.meta.get('registration_choice', None)
-    if registration_choice == EMBARGO:
-        time = parser.parse(draft.approval.meta['embargo_end_date'])
-        embargo = iso8601format(time) if json_safe else time
+    node_url = get_url(draft)
+
+    embargo = get_embargo(draft, json_safe)
+
+    if draft.is_approved:
+        status = get_approval_status(draft)
```

Comment:
Minor, but why not have this helper determine approval status in the other cases as well?